kf/bpf.go

```diff
@@ -1104,10 +1104,19 @@ func (b *BPF) RemoveMapFiles(ifaceName string) error {
 		} else {
 			mapFilename = filepath.Join(b.hostConfig.BpfMapDefaultPath, ifaceName, k)
 		}
+		if !strings.HasPrefix(mapFilename, b.hostConfig.BpfMapDefaultPath) {
+			return fmt.Errorf("malicious mapFilename path")
+		}
 		if err := v.Unpin(); err != nil {
-			return fmt.Errorf("BPF program %s prog type %s ifacename %s map %s:failed to pin the map err - %#v",
+			return fmt.Errorf("bpf program %s prog type %s ifacename %s map %s:failed to pin the map err - %#v",
 				b.Program.Name, b.Program.ProgType, ifaceName, mapFilename, err)
 		}
+		if fileExists(mapFilename) {
+			log.Warn().Msgf("unpinning not able to remove map file : %v", mapFilename)
```

Comment:
Update the log message to "Removing the Map file, as unpinning not able to remove map file"